examples/dataset.py

```diff
@@ -0,0 +1,69 @@
+# Copyright © 2021 United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration.  All Rights Reserved.
+
+"""
+Example downloading and using a NASA prognostics dataset.
+
+In this example, a battery dataset is downloaded from the NASA PCoE data repository. This dataset is then accessed and plotted. 
+"""
+
+DATASET_ID = 1
+
+def run_example():
+    # Step 1: Download and import the dataset for a single battery
+    # Note: This may take some time
+    from prog_models.datasets import nasa_battery
+    print('Downloading... ', end='')
+    (desc, data) = nasa_battery.load_data(DATASET_ID)
+    print('done')
+
+    # We recommend saving the dataset to disk for future use
+    # This way you dont have to download it each time
+    import pickle
+    pickle.dump((desc, data), open(f'dataset_{DATASET_ID}.pkl', 'wb'))
+
+    # Step 2: Access the dataset description
+    print(f'\nDataset {DATASET_ID}')
+    print(desc['description'])
+    print(f'Procedure: {desc["procedure"]}')
+
+    # Step 3: Access the dataset data
+    # Data is in format [run_id][time][variable]
+    # For the battery the variables are 
+    #    0: relativeTime (since beginning of run)
+    #    1: current (amps)
+    #    2: voltage
+    #    3: temperature (°C)
+    # so that data[a][b, 3] is the temperature at time index b (relative to the start of the run) for run 1
```

Comment:
Should line 37 be ... # so that data[a][b][3] is the temperature at time index b (relative to the start of the run) for run a ... The changes here are 1) the three brackets instead of two with the comma to match format [run_id][time][variable], and 2) the "run 1" in the original comment is changed to "a" to match the notation ... if I'm understanding correctly!